uberfire-security/uberfire-security-management/uberfire-security-management-wildfly/uberfire-security-management-wildfly10/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
au contraire.. isn't this "expected prolog" http://www.w3schools.com/xml/xml_syntax.asp